.ci/common.sh

```diff
@@ -48,7 +48,15 @@ check_core_dumps() {
 }
 
 setup_deps() {
+	sudo pip --upgrade pip
 	sudo pip install neovim
+
+	# For pip3
+	# https://github.com/travis-ci/travis-ci/issues/1528 
+	curl http://python-distribute.org/distribute_setup.py | sudo python3
```

Comment:
You should probably use `curl -Ss`, like is done in the rest of the file, although I'm not sure it would still error out correctly from a pipe.